lib/controller.rb

```diff
@@ -6,6 +6,16 @@
 
 class WeekThreeAssessment < Sinatra::Base
 
-  # Your code will go here.
+  get '/' do
+    erb :index
+    @name="Casper"
+  end
+
+  get '/name/:Casper' do
+    user = User.create(:name => 'Casper')
+    user.save
+      @namn1 = User.all
+    erb :name
+  end
```

Comment:
`:Casper` here is a *variable*. Instead of `:Casper`, you want to use `/name/:name`. Then in line 15, you can use `user = User.create(name: name)`. (Also, don't use the "hash rocket" notation).